DatadogTrace/Sources/Span/SpanEventEncoder.swift

```diff
@@ -247,5 +247,8 @@ internal struct SpanEventEncoder {
             let metaKey = "meta.\($0.key)"
             try container.encode($0.value, forKey: DynamicCodingKey(metaKey))
         }
+
+        let metaDdPTid = "meta._dd.p.tid"
+        try container.encode(span.traceID.idHiHex, forKey: DynamicCodingKey(metaDdPTid))
```

Comment:
change-request/ This method is made for encoding "custom meta" fields using dynamic coding keys:
```swift
/// Encodes `meta.*` attributes coming from user
private func encodeCustomMeta(_ span: SpanEvent, to container: inout KeyedEncodingContainer<DynamicCodingKey>) throws {
```

There is another one, declared above, which encodes default tags using type-safe keys. We should prefer it for this use case. It will add more clarity and avoid allocating string literal each time:
```swift
/// Encodes default `meta.*` attributes
private func encodeDefaultMeta(_ span: SpanEvent, to container: inout KeyedEncodingContainer<StaticCodingKeys>) throws {
```

Reply:
sure. fixed.